internal/dot/graph.go

```diff
@@ -20,6 +20,28 @@
 
 package dot
 
+import (
+	"fmt"
+)
+
+// Ctor encodes the edges in a graph. It includes information like the name
```

Comment:
nit: A simple description of the concept of an `edge` (in this context) might help here.